lucene/test-framework/src/java/org/apache/lucene/analysis/BaseTokenStreamTestCase.java

```diff
@@ -125,7 +125,7 @@ public void reflectWith(AttributeReflector reflector) {
   //     arriving to pos Y have the same endOffset)
   public static void assertTokenStreamContents(TokenStream ts, String[] output, int startOffsets[], int endOffsets[], String types[], int posIncrements[],
```

Comment:
maybe we should not change this method signature and just add another one, so we do not need to change unrelated files like MinHashFilter?

Reply:
Done, I think what was originally submitted was a bit easier than sorting out all the parameters at the time, but yes nicer to minimize.